README.md

```diff
@@ -67,6 +67,30 @@ cd notification-daemon-mac-py
 ./notify.py
 ```
 
+### Docker
+
+An experimental Docker image can be built for Pantalaimon, primarily for use in bots.
+
+```bash
+docker build -t pantalaimon .
+# Create a pantalaimon.conf before running. The directory mentioned in the
+# volume below is for where Pantalaimon should dump some data.
+docker run -it --rm -v /path/to/pantalaimon/dir:/data -p 8008:8008 pantalaimon
+```
+
+An example `pantalaimon.conf` for Docker is:
+```conf
+[Default]
+LogLevel = Debug
+SSL = True
+
+[local-matrix]
+Homeserver = https://matrix.org
+ListenAddress = 0.0.0.0
+ListenPort = 8008
+SSL = False
```

Comment:
You might want to set `UseKeyring` to False/No and `IgnoreVerification` to True/Yes here.

Reply:
I didn't realize those were options - will try them.

Reply:
they seem to do roughly the right thing, so will use them.